tests/foreman/installer/test_installer.py

```diff
@@ -1355,10 +1355,12 @@ def install_satellite(satellite, installer_args, enable_fapolicyd=False):
         snap=settings.server.version.snap,
     )
     if enable_fapolicyd:
-        assert (
-            satellite.execute('dnf -y install fapolicyd && systemctl enable --now fapolicyd').status
-            == 0
-        )
+        if satellite.execute('rpm -q satellite-maintain').status == 0:
+            # Installing the rpm on existing sat needs sat-maintain perms
+            cmmd = 'satellite-maintain packages install -y fapolicyd'
```

Comment:
This is wrong, there's won't be any satellite related packages installed on the rhel machine. It has to be done through dnf. It's done later in the setup
```suggestion
```